sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Added missing marray relational functions: `any`, `all`, `bitselect`; fixed scalar `select` relational function; aligned scalar `abs` integer function with SYCL 2020 and fixed math functions which take `multi_ptr` argument. [8b0b210a95db]
+- Added `operator[]` and `element_type` to swizzle vec to align with SYCL 2020. [9050d3325f2c]
+- Fixed buffer range in `atomic_memory_order_acq_rel`. [9fb49f757f04]
+- Started to throw for invalid `global_work_size` query. [61433888d4b5]
+- Fixed the Level Zero backend to report events as submitted, not running, until they are completed. [502e0fdd3f3c]
+- Allowed raw send ESIMD API to use non-standard types like `sycl::half`. [8022c6a4f6ec]
+- Started to throw for invalid `info::kernel::num_args query` to comply with SYCL 2020. [93e8ca3ed2eb]
+- Updated `group_broadcast` to support `vec` types. [09ae6434f55f]
+- Fixed reductions to avoid implicit `atomic64` requirements. [e11b35835722]
+- Added a partial profiling workaround for acc devices because queue profiling is no longer supported for OpenCL version < 2.1 after recent changes regarding command submit profiling info. [097d21c33599]
+- Fixed program build API for the Level Zero backend to retain build-log when program build failed. [7d9f5ac5e5da]
+- Implemented missing std::hash specializations for local_accessor and host_accessor to comply with SYCL 2020. [919246bacb82]
+- Fixed `global_work_size` kernel query. [d666b95af56a]
+- Fixed incorrect `sycl::vec<bool, N>` constructor behavior. [45aa49bb3f06]
+- Fixed `weak_object` for `host_accessor` and `stream`. [225b3ecad28d]
+- Fixed incorrect behaviors in some operations using `sycl::vec<bool, N>`. [61f6ce4579b7]
+- Fixed integration footer for `specialization_id`. [dd96dfaf12da]
+- Fixed compilation break occurring when bfloat16 constructor is used in a kernel. [bc063ac7a9e1]
+- Fixed handling of subgroup info queries. [2408035da9c9]
+- Added missing support for `target::host_task` specialised accessor constructor using `mode_target Tag`. [c01d8824b1aa]
+- Fixed identityless reductions with unwritten reducers. [cb15f33e861a]
+- Fixed dangling pointer issue in xpti tracing. [8a49660d4b44]
+- Fixed PI event leak in memcpy2d device-host fallback. [24ec33c39c85]
+- Fixed `weak_object` and `owner_less` for `device` objects. [b9b85136d492]
+- Added `noexcept` specifier to `vec::byte_size` to comply with SYCL 2020. [cb6c0580dc30]
+- Fixed undefined behaviour in vector printf specifier. [d190de65a4e1]
+- Fixed accessor fill for unsupported patterns. [7f49367d234b]
+
+### Documentation
+- Fixed link anchors in [`EnvironmentVariables`](doc/EnvironmentVariables.md) documentation. [ba1fc2e68d04]
+- Fixed InlineAsm example in [`sycl_ext_intel_esimd`](doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md). [5ff169559712]
+
+## API/ABI breakages
```

Comment:
Return type changes mentioned in bugfix section are also API breaking. Also, promotion of APIs hidden under `SYCL2020_CONFORMANT_APIS` was an API breaking change

Reply:
Thank you, moved those to API breaking section.